src/managers/saved-runs-manager.js

```diff
@@ -11,13 +16,15 @@ var injectFiltersFromSession = require('./strategy-utils').injectFiltersFromSess
 var SavedRunsManager = function (config) {
     var defaults = {
         /**
-         * If set, will only pull runs from current group by default
+         * If set, will only pull runs from current group. Defaults to `true`.
          * @type {Boolean}
          */
         scopeByGroup: true,
 
         /**
-         * If set, will only pull runs from current user by default
+         * If set, will only pull runs from current user. Defaults to `true`.
```

Comment:
I'd add a note for why you'd want to change the default; i.e. for e.g. if you want all the users in your group to share the same set of saved runs.